superset/db_engine_specs/gsheets.py

```diff
@@ -87,6 +105,50 @@ def extra_table_metadata(
 
         return {"metadata": metadata["extra"]}
 
+    @classmethod
+    def build_sqlalchemy_uri(cls) -> str:
+        # All gsheets return just the engine in the format of the uri,
+        # the above are temporary for csi
+        if not cls:
```

Comment:
remove this we dont need to check the cls method it come built int

Reply:
I added this because you can't have a method without a variable in it. 

Or could I do `def build_sqlalchemy_uri:` ?